### PR TITLE
add TaskCards and aggregate on my tasks screen

### LINKED ISSUE #349: Add task card
grafik

### PR DESCRIPTION
**Pull request by DasProffi**
## Which issues does this pull request close?
closes #349

### CODE REVIEW

File: apps/tasks/lib/components/task_card.dart [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+
+class TaskCard extends StatelessWidget {
+  final TaskWithPatient task;
+  final EdgeInsetsGeometry? margin;
+  final double borderRadius;
+
+  const TaskCard({super.key, required this.task, this.margin, this.borderRadius = borderRadiusMedium});
+
+  String getDueText(BuildContext context) {
+    String result = context.localization!.dueIn;
```
Comment (florian-sabonchi) [inline apps/tasks/lib/components/task_card.dart:15]: Please remove the unnecessary else

Comment (florian-sabonchi) [reply apps/tasks/lib/components/task_card.dart:15]: and i think it would make sense use a switch statement instead

Comment (florian-sabonchi) [reply apps/tasks/lib/components/task_card.dart:15]: `class MyLocalizations {
  static String minutes(int count) => Intl.plural(
    count,
    one: '1 minute',
    other: '$count minutes',
    name: 'minutes',
  );
}`

File: apps/tasks/lib/components/task_card.dart [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+import 'package:tasks/dataclasses/task.dart';
+
+class TaskCard extends StatelessWidget {
+  final TaskWithPatient task;
+  final EdgeInsetsGeometry? margin;
+  final double borderRadius;
+
+  const TaskCard({super.key, required this.task, this.margin, this.borderRadius = borderRadiusMedium});
+
+  String getDueText(BuildContext context) {
```
Comment (florian-sabonchi) [inline apps/tasks/lib/components/task_card.dart:14]:
```suggestion
   String getDueText(BuildContext context) {
    String result = context.localization!.dueIn;
    if (task.dueDate == null) {
      return "$result -";
    }

    if (task.isOverdue) {
      return context.localization!.overdue;
    }

    if (task.inNextHour) {
      return "$result ${context.localization!.nMinutes(task.remainingTime.inMinutes)}";
    }

    if (task.inNextTwoDays) {
      return "$result ${context.localization!.nHours(task.remainingTime.inHours)}";
    }

    return "$result ${context.localization!.nDays(task.remainingTime.inDays)}";
  }
```

**Review by florian-sabonchi — CHANGES_REQUESTED** [review]

**Review by MaxSchaefer — APPROVED** [review]

**Review by florian-sabonchi — APPROVED** [review]

**Pull request merged**